src/scss/components/_button.scss

```
@@ -137,7 +159,9 @@ $button-padding-h: $grid-margin-2;
     $color-border-hover: var(--of-button-danger-hover-color-border),
     $color-active: var(--of-button-danger-active-fg),
     $color-background-active: var(--of-button-danger-active-bg),
-    $color-border-active: var(--of-button-danger-active-color-border)
+    $color-border-active: var(--of-button-danger-active-color-border),
+    $color-border-focus: var(--of-button-danger-focus-color-border),
```

let's fall back to the `danger` token here from Utrecht

```suggestion
    $color-border-focus: var(--of-button-danger-focus-color-border, var(--utrecht-button-primary-action-danger-focus-border-color)),
```